codecs/opus/src/test/java/org/restcomm/media/codec/opus/OpusCodecTest.java

```diff
@@ -70,26 +79,61 @@ public void tearDown() throws Exception {
      * Test of process method, for Encoder and Decoder.
      */
     @Test
-    public void testCodec() {
-    	OpusJni opus = new OpusJni();
-    	opus.setOpusObserverNative(this);
-    	opus.sayHelloNative();
+    public void testCodec() throws Exception {
+    	
+    	boolean testPassed = false;
+    	
+    	try {
+    		final int packetSize = 480;
+    		File outputFile = File.createTempFile("opustest", ".tmp");
+	        FileInputStream inputStream = new FileInputStream("src\\test\\resources\\test_sound_mono_48.pcm");
+	        FileOutputStream outputStream = new FileOutputStream(outputFile, false);
+	    	
+	    	OpusJni opus = new OpusJni();
+	    	opus.setOpusObserverNative(this);
```

Comment:
Create a Mockito.mock for OpusJni.Observer

Reply:
Done